luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/AbridgedLogicalTableGroupFactory.java

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.config.luthier.factories;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.yahoo.bard.webservice.config.luthier.Factory;
+import com.yahoo.bard.webservice.config.luthier.LuthierIndustrialPark;
+import com.yahoo.bard.webservice.config.luthier.LuthierValidationUtils;
+import com.yahoo.bard.webservice.data.config.LogicalTableGroup;
+import com.yahoo.bard.webservice.data.config.names.ApiMetricName;
+import com.yahoo.bard.webservice.data.config.names.LogicalTableName;
+import com.yahoo.bard.webservice.data.dimension.Dimension;
+import com.yahoo.bard.webservice.data.metric.MetricDictionary;
+import com.yahoo.bard.webservice.data.time.DefaultTimeGrain;
+import com.yahoo.bard.webservice.data.time.ZonedTimeGrain;
+import com.yahoo.bard.webservice.table.LogicalTable;
+import com.yahoo.bard.webservice.table.PhysicalTable;
+import com.yahoo.bard.webservice.table.TableGroup;
+import com.yahoo.bard.webservice.table.TableIdentifier;
+import org.joda.time.DateTimeZone;
+
+import java.util.LinkedHashSet;
+import java.util.Locale;
+
+/**
+ * A factory that is used by default to support LogicalTableGroup.
```

Comment:
We should include some documentation here about what a LogicalTableGroup is, and why we're building a collection of
LogicalTables from one config source instead of a single LogicalTable from one config source like people might expect.

Reply:
I wrote a doc, please feel free to give it a check.